css/components/modal.css

```diff
@@ -0,0 +1,30 @@
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(58, 49, 49, 0.7);
```

Comment:
use variables

Reply:
Thanks Varun for the review, I have made the suggested changes. 😄